lens/operators.mli

```diff
@@ -27,7 +38,10 @@ module Binary : sig
 
   val to_string : t -> string
 
-  val of_string : string -> t
+  val of_string : string -> (t, Operator_not_found_exception.t) result
+
+  (** Do not use. *)
+  val of_string_exn : string -> t
```

Comment:
Why is this here if we're not allowed to use it?

Reply:
`_exn` functions are library functions useful for writing quick code / unit tests but as the name suggests they can throw exceptions which is not how most of the code I have been writing is supposed to work. This is why most are implemented with `failwith` as I try to (not 100% succesfully, but working on it) to not use any `_exn` functions in the code.

Reply:
Maybe the fact that your intention is to avoid and eventually remove these could be documented by using  `_failwith` as a suffix instead of `_exn`?